doc/source/api.rst

```diff
@@ -5,66 +5,108 @@
 API Reference
 =============
 
-.. currentmodule:: cvpy
+**********
+Annotation
+**********
 
-Biomedical Images
------------------
+Base
+====
+
+Credentials
+-----------
+
+.. currentmodule:: cvpy.annotation.base.Credentials
 
-.. currentmodule:: cvpy.biomedimage.BiomedImage
-    
 .. autosummary::
     :toctree: generated/
 
-    BiomedImage
-    BiomedImage.quantify_sphericity
-    BiomedImage.mask_image
-    BiomedImage.morphological_gradient
+    Credentials
 
-CAS Thread Tuner
-----------------
+Project
+-------
 
-.. currentmodule:: cvpy.utils.CASThreadTuner
+.. currentmodule:: cvpy.annotation.base.Project
 
 .. autosummary::
     :toctree: generated/
 
-    CASThreadTuner
-    CASThreadTuner.tune_thread_count
+    Project
+    Project.as_dict
+    Project.get_annotations
+    Project.post_images
+    Project.resume
+    Project.save
+    Project.to_json
 
+CVAT
+====
 
-CAS Thread Tuner Results
-------------------------
+.. currentmodule:: cvpy.annotation.cvat
 
-.. currentmodule:: cvpy.base.CASThreadTunerResults
+CVAT Project
+------------
+
+.. currentmodule:: cvpy.annotation.cvat.CVATProject
 
 .. autosummary::
     :toctree: generated/
 
-    CASThreadTunerResults
-    CASThreadTunerResults.plot_exec_times
+    CVATProject
+    CVATProject.as_dict
+    CVATProject.from_json
+    CVATProject.get_annotations
+    CVATProject.post_images
+    CVATProject.save
+    CVATProject.resume
+    CVATProject.to_json
+
+CVAT Task
```

Comment:
The end users should never have to create or interact with a CVATTask object. It is automatically created by the CVATProject.post_images() call. So it can be completely removed from the doc.